crates/orderbook/src/arguments.rs

```diff
@@ -280,6 +289,9 @@ impl std::fmt::Display for Arguments {
             "fast_price_estimation_results_required: {}",
             self.fast_price_estimation_results_required
         )?;
+        write!(f, "price_estimation_rate_limiter: ")?;
+        display_option(&self.price_estimation_rate_limiter, f)?;
+        writeln!(f)?;
```

Comment:
It is also okay to do 

```rust
writeln!(
            f,
            "price_estimation_rate_limiter: {:?}",
            self.price_estimation_rate_limiter
        )?;
```

If you are fine with the default Debug implementation of the struct and don't want to impl Display for it.

Reply:
I had that initially but that didn't look very nice.
This is how it looks now with only the configured values displayed:
```
...
fast_price_estimation_results_required: 2
price_estimation_rate_limiter: RateLimitingStrategy{ min_back_off: 60s, max_back_off: 60s, growth_factor: 2.0 }
token_detector_fee_values: Static
...
```